js/articleView.js

```diff
@@ -3,6 +3,8 @@
 (function(module) {
   var articleView = {};
 
+// TODO I need to clean up this code and get rid of the commented code or figure out how to use it. Again, this code relates to the category filters. I would like to get them working. Even though at this point they seem a little useless. As I add more projects to my portfolio I think they will become much more useful.
+
```

Comment:
Getting rid of corpse code will really help you see your code and debug, and add more functionality. You already noted this, so you know what to do there! I suggest commenting throughout so you can really appreciate what your code is doing ( less to think about!), and add to and refactor.